CMakeLists.txt

```diff
@@ -1694,6 +1694,7 @@ check_function_exists("fnmatch"       HAVE_FNMATCH)
 check_symbol_exists("basename"        "${CURL_INCLUDES};string.h" HAVE_BASENAME)  # libgen.h unistd.h
 check_symbol_exists("opendir"         "dirent.h" HAVE_OPENDIR)
 check_function_exists("poll"          HAVE_POLL)  # poll.h
+check_function_exists("realpath"      HAVE_REALPATH)
```

Comment:
I suggest skipping the check for Windows in both cmake/autotools. It's not present and not used, just taking config time. There is a groups for these a few lines later around `sched_yield`.